etl/main.py

```diff
@@ -0,0 +1,212 @@
+"""
+This module comprises the ETL system for the "Cuarto de Milla" project, and is
+in charge of processing Gas Station datasets into a clean structure and then
+insert records into a database
+"""
+
+import os
+import sys
+import random
+import requests
+import traceback
+import numpy as np
+import pandas as pd
+import sqlalchemy as db
+import xmltodict as x2d
+from datetime import timedelta, datetime
+from timeit import default_timer as timer
+from dummy import get_city_and_state_for_location
+
+
+URL_SOURCES = [
+    'https://publicacionexterna.azurewebsites.net/publicaciones/places',
+    'https://publicacionexterna.azurewebsites.net/publicaciones/prices'
+]
+DB_STRING = os.environ['DATABASE_URL']
+DATA_FOLDER = 'data/'
```

Comment:
Please create the variables using absolute routes. 

You can use os library

Reply:
Fixed.